### PR TITLE
Port readme.ptxt to README.md

### PR DESCRIPTION
**Pull request by spnda**
Ported the readme.ptxt file to a markdown file.

All of the text is from the old .ptxt readme. Just the text and content styling has changed. (And I fixed some typos 😉 )
The README.md file will automatically be put up on the main site by GitHub.
You can view the new MarkDown File [here](https://github.com/ThePNDA/OpenGFX/blob/master/docs/README.md).

**Please check if I missed anyone in the credits section!**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+| Roujin            | Manuel Wolf           | N/A                                           |
+| Red*Star          | David Krebs           | N/A                                           |
+| Raumkraut         | Mel Collins           | N/A                                           |
+| Purno             | Mark Leppen           | N/A                                           |
+| planetmaker       | Ingo van Borstel      | German                                        |
+| PikkaBird         | David Dallaston       | N/A                                           |
+| Phreeze           | N/A                   | Luxemburgean                                  |
+| Pekkape01         | N/A                   | Swedish                                       |
+| PaulC             | Paul Charlesworth     | N/A                                           |
+| orduge            | Owen Ridge            | N/A                                           |
```
Comment (JGRennison) [inline docs/README.md]: Owen's handle and surname are both misspelled here, they're correct in the original file

Comment (ThePNDA) [reply docs/README.md]: The new commit https://github.com/OpenTTD/OpenGFX/pull/2/commits/449cad069b3c403059ab242f9d5c54f114039768 has fixed his handle and surname. Thanks for pointing this out!

File: docs/README.md [hunk 2]
```diff
@@ -168,95 +174,95 @@ this program; if not, write to the
...
-* Portuguese:                 vesgo
-* Russian:                    George
-* Scotish Gaelic:             GunChleoc
-* Serbian:                    stravagante
-* Slovak:                     Tenebrae
-* Spanish:                    SilverSurveferZzZ
-* Swedish:                    Zuu, Pekkape01
-* Tamil:                      Aswn
-
+| Name              | Realname              | Translations                                  |
```
Comment (planetmaker) [inline docs/README.md]: When unifying the contributors and translator tables, the information goes missing that a person can have contributed both (code / graphics) and translations. Just one example where this gets lost is Zuu.

I'd like to keep this information separate as it is really a much different kind of level of involvement. Not sure about whether it's actually used in any way like this: the original idea behind an alphabetical order of languages is to look who is responsible for a translation and contact that person when you speak that language...

File: docs/README.md [hunk 3]
```diff
@@ -74,20 +77,21 @@ The baseset directories are:
...
+
 
 ## 4.0 Reporting bugs and contributing
 
 ### 4.1 Reporting bugs
 
-If you spot any graphical bugs or glitches in the available graphics, please let us know preferrably via our [issue tracker](https://github.com/OpenTTD/OpenGFX/issues) or via the OpenGFX release topic on the [TT-Forums](http://www.tt-forums.net/viewtopic.php?f=36&t=40162).
-Please make sure that you're using the latest available version before reporting a bug. You can check the [issue tracker](https://github.com/OpenTTD/OpenGFX/issues) to see if the bug you've found is already reported (or fixed!).
+If you spot any graphical bugs or glitches in the available graphics, please let us know preferrably via our [issue tracker](https://dev.openttdcoop.org/projects/opengfx/issues) or via the OpenGFX release topic on the [TT-Forums](http://www.tt-forums.net/viewtopic.php?f=36&t=40162).
```
Comment (planetmaker) [inline docs/README.md]: The issue tracker etc probably should now link to GitHub as well. If we move the project, we move it completely

File: docs/README.md (hunk 2 above)
Comment (spnda) [reply docs/README.md]: So having them seperate as before is better in your opinion?

File: docs/README.md (hunk 3 above)
Comment (spnda) [reply docs/README.md]: I wanted to use the GitHub issue tracker initially, though due to the issue tracker not being set up on this repo I decided to put the openttdcoop.org's issue tracker in. I can sure change this, but currently visiting https://github.com/OpenTTD/OpenGFX/issues redirects to https://github.com/OpenTTD/OpenGFX/pulls.

Comment (planetmaker) [reply docs/README.md]: Oh. That actually is an oversight on my part. Let's quickly activate the issues. Thanks for that pointer!

File: docs/README.md (hunk 2 above)
Comment (planetmaker) [reply docs/README.md]: I tend do answer 'yes'. Unless you want to add a separate column or two with Graphics and Code... but that doesn't do it justice either. And docs/authorreview.csv is not exactly complete (yet much better than anything, if you're interested in who did what. Integrating that at that level of detail in this commit might be asking a bit too much)

File: docs/README.md (hunk 3 above)
Comment (spnda) [reply docs/README.md]: Also mentioning a site which can also be moved from the openttcoop.org's site to GitHub:
There is a Wiki section available in a GitHub repository, which could be activated. 👍

**Comment by spnda** [comment]
The new commit https://github.com/OpenTTD/OpenGFX/pull/2/commits/8f4d23c73bbcacbe99b5d092e4c322b05be55144 resolves both issues stated. All changes necessary have been fulfilled.

File: docs/README.md (hunk 3 above)
Comment (planetmaker) [reply docs/README.md]: Yes... what purpose would this wiki serve, though? Anything more general baseset probably is better placed in the OpenTTD wiki

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+The OpenGFX source is available in a Mercurial repository or as gzip'ed tarball. You can do an anonymous checkout from http://hg.openttdcoop.org/opengfx, e.g. using `hg clone http://hg.openttdcoop.org/opengfx` or obtain the tarball from the [release page](https://github.com/OpenTTD/OpenGFX/releases).
+
+### 4.3 Building OpenGFX
+
+Prerequisites to building OpenGFX:
+- gcc (the pre-processor is needed)
+- [NML 0.4.2](http://bundles.openttdcoop.org/nml/) (default / development branch)
+- grfid from the [grfcodec package](http://www.openttd.org/download-grfcodec)
+- some gnu utils: `make`, `cat`, `sed`, `awk` and you might additionally want a text editor of your choice and a graphics programme suitable to handle palettes.
+- [Mercurial](http://mercurial.selenic.com/wiki/Download?action=show&redirect=BinaryPackages) (only when not building from a tarball)
```
Comment (planetmaker) [inline docs/README.md]: This probably should become git now

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+- The variable which supplies the install path changed for the sake of consistency and better readability to `INSTALL_DIR`. The old `INSTALLDIR` still works but is deprecated.
+
+### 4.5 Note on the .xcf and .psd files
+
+The repository contains a few `.xcf2png` files which indicate which png files can be generated from the source `.xcf` or `.psd` files. This will only be used, if GIMP is found. Calling `maintainer-clean` will delete the png files which can be re-generated from a `.xcf` or `.psd` file.
+
+## 5.0 License
+
+OpenGFX Graphics Base Set for OpenTTD
+Copyright (C) 2007-2016 by the OpenGFX team (see [credits section](#60-credits) below)
```
Comment (planetmaker) [inline docs/README.md]: 2019?

**Comment by spnda** [comment]
Commit https://github.com/OpenTTD/OpenGFX/pull/2/commits/6c0e42776aa5a0c1227c089effaa56bd857e944d has fixed both of these issues.

**Pull request merged**